core/src/main/kotlin/com/malinskiy/marathon/execution/queue/TestAction.kt

```diff
@@ -1,8 +1,5 @@
 package com.malinskiy.marathon.execution.queue
 
-import com.malinskiy.marathon.device.DeviceInfo
-import com.malinskiy.marathon.execution.TestResult
-
 sealed class TestAction {
-    data class SaveReport(val deviceInfo: DeviceInfo, val testResult: TestResult) : TestAction()
+    object Conclude : TestAction()
```

Comment:
Maybe use most common name for that action. `End` or `Complete`

Reply:
Or `Finish`?

Reply:
One of the hardest problems in CS :) Let's use 'Complete'